css/properties.json

```diff
@@ -5315,6 +5315,22 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/line-height"
   },
+  "line-height-step": {
+    "syntax": "<length>",
+    "media": "visual",
+    "inherited": true,
+    "animationType": "discrete",
+    "percentages": "no",
+    "groups": [
+      "CSS Fonts"
+    ],
+    "initial": "0",
+    "appliesto": "blockContainers",
+    "computed": "absoluteLength",
+    "order": "perGrammar",
+    "status": "experimental",
+    "mdn_url": "https://developer.mozilla.org/docs/Mozilla/CSS/line-height-step"
```

Comment:
I got a 404 with this. The correct one seems to be: https://developer.mozilla.org/docs/Web/CSS/line-height-step

Reply:
Good catch, thank you! Fixed!